### ISSUE TITLE
[joss][ui] control points contrast

### ISSUE THREAD

**Issue by anibalsolon** [neuroanatomy/thresholdmann#35]
Part of: https://github.com/openjournals/joss-reviews/issues/6336

Extracted from the main manuscript:
![image](https://github.com/neuroanatomy/thresholdmann/assets/562525/7552b05f-5f9b-471f-8d96-2e419f89e7c5)

And from the main example:
![image](https://github.com/neuroanatomy/thresholdmann/assets/562525/135c75ae-2bf7-4750-b35e-2fdc1b64ccf0)

I find that, for some cases, control points are hard to see. According to a contrast check, it would fail the WCAG compliance.
https://webaim.org/resources/contrastchecker/

![image](https://github.com/neuroanatomy/thresholdmann/assets/562525/2719c644-c769-481f-8fe1-85e65de3d8c0)

Although much hard to provide a full accessibility, I believe improving the points visibility would be a great modification to the tool. For example, adding a lighter border to the point or making it blink upon clicking on it on the right panel, or even drawing the control point in the canvas, allowing it to compute the most optimal contrasting color.

![image](https://github.com/neuroanatomy/thresholdmann/assets/562525/cafeba41-ac0a-4659-8f7e-a703fb936773)

**Comment by katjaq** [comment]
Thank you for this feedback and the linked resource which will keep in mind also for future applications :)
We implemented a thin solid white border around each control point for better visibility in commit #[4f4185b](https://github.com/neuroanatomy/thresholdmann/commit/4f4185bb7e31d55c63be2d355e09a311594314ff) as suggested by the reviewer.
